### PR TITLE
configs: cleanup invenio.cfg

### PR DESCRIPTION
**Pull request by alejandromumo**
closes https://github.com/zenodo/zenodo-rdm/issues/524

### CODE REVIEW

File: invenio.cfg [hunk 1]
```diff
@@ -550,3 +550,9 @@ OPENAIRE_DIRECT_INDEXING_ENABLED = True
 """Enable sending published records for direct indexing at OpenAIRE."""
 
 RDM_RECORDS_SERVICE_COMPONENTS = DefaultRecordsComponents + [OpenAIREComponent]
+
+MAIL_DEFAULT_SENDER = "Zenodo <info@zenodo.org>"
```
Comment (slint) [inline invenio.cfg:554]: @lnielsen what's the `noreply@cern.ch` address equivalent we should be using for Zenodo?

Comment (lnielsen) [reply invenio.cfg:572]: I'm checking with Emmanuel.

Comment (lnielsen) [reply invenio.cfg:572]: We can use ``noreply@zenodo.org``

File: invenio.cfg [hunk 2]
```diff
@@ -567,4 +567,10 @@ USERS_RESOURCES_ADMINISTRATION_ENABLED = True
...
 COMMUNITIES_ADMINISTRATION_DISABLED = False
-"""Enable communities administration."""
\ No newline at end of file
+"""Enable communities administration."""
+
+MAIL_DEFAULT_SENDER = "Zenodo <info@zenodo.org>"
+"""Default e-mail address sender."
+
+REST_ENABLE_CORS = True
+"""Enable CORS validation.""""
```
Comment (zzacharo) [inline invenio.cfg:576]: Should we evaluate the [CSP rules](https://github.com/zenodo/zenodo-rdm/pull/542/files#diff-35ea7d0c9f0d3771b33ec0ac671404047bdff444801fcf084c39437c21107ebfR85-R86) as well? Especially, the fact that we allow `unsafe-inline` and `unsafe-eval`?

File: invenio.cfg (hunk 1 above)
Comment (alejandromumo) [reply invenio.cfg]: Modified the config for Zenodo-RDM 👍

**Pull request merged**